backend/src/utils/notebookUtils.ts

```diff
@@ -277,7 +278,7 @@ export const assembleNotebook = async (
           enableServiceLinks: false,
           containers: [
             {
-              image: imageUrl,
+              image: ' ',
```

Comment:
Same here -- intended a single space string?

Reply:
Yes, it is part of the image stream admission plugin together with the annotation above. When one of the core Kubernetes resources contains both a pod template and this annotation, OpenShift Container Platform attempts to update the object by using the image currently associated with the image stream tag that is referenced by trigger. The update is performed against the fieldPath specified in the annotation. On initial creation of the statefulset derived from the notebook CR, this only works when either the image-field is completely left out or set to a single space string. https://docs.openshift.com/container-platform/4.10/openshift_images/triggering-updates-on-imagestream-changes.html#images-triggering-updates-imagestream-changes-kubernetes-about_triggering-updates-on-imagestream-changes.

Reply:
With your permissions for DSG already being correct, you can also test this with a statefulset in a different namespace than the imagestream, that is why the trigggers annotation also has a namespace-section in its value.

Reply:
> Same here -- intended a single space string?

can be anything, the annotation and the openshift image policy admission plugin auto-fill-in the correct image value.

Reply:
That's far nicer for the code, thanks @shalberd